FirebaseCore/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Firebase 11.4.1
+- [fixed] CocoaPods only release to revert breaking change in
+  `FirebaseCoreExtension` SDK. (https://github.com/firebase/flutterfire/issues/13527)
```

Comment:
nit: Consider changing to `(#13942)` (which has a reference to the Flutter issue.

Reply:
thanks!